src/Player.js

```diff
@@ -1,3 +1,7 @@
+import Game from './Game.js';
+import domUpdates from './domUpdates.js';
+
+
 class Player {
   constructor(name, score = 0) {
     this.name = name;
```

Comment:
Don't forget that they'll need 2 scores! (Round & Total)
You can always add it later in a function, vs hardcoding the value at the start.